source/jormungandr/jormungandr/autocomplete/abstract_autocomplete.py

```diff
@@ -54,7 +54,7 @@ def get(self, query, instances):
         pass
 
     @abstractmethod
-    def get_by_uri(self, uri, request_id=None, instances=None, current_datetime=None):
+    def get_by_uri(self, uri, request_id, instances=None, current_datetime=None):
```

Comment:
Are you sure to delete default value? I mean we have request_id in all uses of this function?